python/cuml/test/dask/test_tsvd.py

```diff
@@ -35,10 +35,10 @@ def test_pca_fit(data_info, input_type, client):
 
     from cuml.dask.datasets import make_blobs
 
-    X, _ = make_blobs(n_samples=nrows,
-                      n_features=ncols,
+    X, _ = make_blobs(n_samples=int(nrows),
```

Comment:
Maybe just fix this in the stress_param? So: `stress_param([int(9e6), 5000, 30]` - seems more direct than passing in a float to the test then fixing it later